src/plugins/intel_npu/tools/protopipe/src/simulation/ov_layers_reader.cpp

```diff
@@ -128,6 +128,18 @@ static void cfgOutputPostproc(ov::preprocess::PrePostProcessor& ppp, const std::
     }
 }
 
+static void cfgPartialShapes(const std::shared_ptr<ov::Model>& model, 
```

Comment:
`cfgReshape` would be enough

Reply:
Well "PartialShapes" is the name of the parameters the reshape is receiving. This is the name the openvino guys called them. I am configuring the parameters not the function.

I will change the function name as requested

Reply:
Done